src/content/docs/crowdin/getting-started/comparing-crowdin-and-crowdin-enterprise.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Comparing Crowdin and Crowdin Enterprise
+description: Learn the key differences between Crowdin and Crowdin Enterprise
+slug: comparing-crowdin-and-crowdin-enterprise
```

Comment:
I would prefer to put this article in the Enterprise section of the docs.

Then, add some references to this article in:

- https://support.crowdin.com/enterprise/introduction/#what-is-crowdin-enterprise - at the end of section
- https://support.crowdin.com/introduction/#what-is-crowdin
- _Crowdin Enterprise_ section on the main page (remove the Enterprise Project Setup)